seed/static/seed/js/controllers/column_settings_controller.js

```diff
@@ -109,18 +160,36 @@ angular.module('BE.seed.controller.column_settings', [])
         });
       };
 
-      // Matching Criteria sorting
-      $scope.column_sort = 'default';
-      $scope.matching_criteria_sort = function() {
-        if ($scope.column_sort === 'default') {
-          $scope.columns = _.sortBy($scope.columns, 'is_matching_criteria');
-          $scope.column_sort = 'is_matching_criteria';
-        } else if ($scope.column_sort === 'is_matching_criteria') {
+      // Table Sorting
```

Comment:
These changes involved a refactor of the previous matching criteria sorting. The columns are now presorted by ID and this state is considered "default".